topics/proteomics/tutorials/neoantigen-7-hla-binding-novel-peptides/tutorial.md

```diff
@@ -0,0 +1,463 @@
+>    - *"With following condition"*: `c20=='Yes'`
+>
+>
+>
+{: .hands_on}
+
+
+### Annotating peptides based on their binding affinity
+
+The peptides, including both strong and weak binders, are annotated with their respective HLA types.
+
+ > <hands-on-title> Annotating weak binder peptides </hands-on-title>
+>
+> 1. {% tool [Query Tabular](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.3.2) %} with the following parameters:
+>    - In *"Database Table"*:
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `table` (output of **Table Compute** {% icon tool %})
+>            - In *"Table Options"*:
+>                - *"Use first line as column names"*: `Yes`
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `out_file1` (output of **Filter** {% icon tool %})
+>    - *"SQL Query to generate tabular output"*:
+>   ``` sql
+>   SELECT * FROM t1
+>   WHERE t1.icore IN (SELECT c1 FROM t2)
+>   ```
+>    - *"include query result column headers"*: `Yes`
+>    - In *"Additional Queries"*:
+>        - In *"SQL Query"*:
+>            - {% icon param-repeat %} *"Insert SQL Query"*
+>                - *"SQL Query to generate tabular output"*:
+>              ``` sql
+>              SELECT icore
+>              FROM t1
+>              WHERE t1.icore IN (SELECT c1 FROM t2)
+>              ORDER BY icore
+>              ```
+>                - *"include query result column headers"*: `No`
+>
+>
+{: .hands_on}
+
+
+> <hands-on-title> Annotating strong binder peptides </hands-on-title>
+>
+> 1. {% tool [Query Tabular](toolshed.g2.bx.psu.edu/repos/iuc/query_tabular/query_tabular/3.3.2) %} with the following parameters:
+>    - In *"Database Table"*:
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `table` (output of **Table Compute** {% icon tool %})
+>            - In *"Table Options"*:
+>                - *"Use first line as column names"*: `Yes`
+>        - {% icon param-repeat %} *"Insert Database Table"*
+>            - {% icon param-file %} *"Tabular Dataset for Table"*: `out_file1` (output of **Filter** {% icon tool %})
+>    - *"SQL Query to generate tabular output"*:
+>   ``` sql
+>   SELECT *
+>   FROM t1
+>   WHERE t1.icore IN (SELECT c1 FROM t2)
+>   ```
+>    - *"include query result column headers"*: `Yes`
+>    - In *"Additional Queries"*:
+>        - In *"SQL Query"*:
+>            - {% icon param-repeat %} *"Insert SQL Query"*
+>                - *"SQL Query to generate tabular output"*:
+>              ``` sql
+>              SELECT icore
+>              FROM t1
+>              WHERE t1.icore IN (SELECT c1 FROM t2)
+>              ORDER BY icore
+>              ```
+>                - *"include query result column headers"*: `No`
+>
+>
+{: .hands_on}
```

Comment:
It feels like this tutorial ends a bit abruptly. Students are asked to perform a step, but nothing is done with the output, results is not discussed or looked at etc. Would it make sense to add either some text here or a question about the output? Especially since this is the final step of a 7-tutorial journey.

Reply:
Oh no..I think I missed writing this..I will add this